powersimdata/design/transmission/substations.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+
+
+def calculate_substation_capacity(grid):
+    """For each substation in a grid, calculate the total substation transmission
+    capacity (in a transport model, ignoring power flow).
+
+    :param powersimdata.input.grid.Grid grid: a grid instance.
+    :return: (*pandas.Series*) -- index is substation IDs, value are total transmission
+        capacity (MW).
+    """
+    # Get new branch data frame with 'from_sub' and 'to_sub' columns
+    branch = grid.branch.assign(
+        from_sub=grid.branch.from_bus_id.map(grid.bus2sub.sub_id),
+        to_sub=grid.branch.to_bus_id.map(grid.bus2sub.sub_id),
+    )
```

Comment:
should we use `from_sub_id` and `to_sub_id` to mirror from_bus_id` and `to_bus_id`?

Reply:
We could. It would cause more line wrapping in the subsequent Series constructions, but it's not like this function is very long anyway.

Reply:
Done. No extra line wrapping due to @BainanXia's simplification.